README.md

```diff
@@ -655,38 +746,63 @@
     });
     ```
 
-  - Place 1 space before anonymous function parenthesis.
+  - Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
 
     ```javascript
-	// bad
```

Comment:
I removed for consistency's sake because that's what I'm used to.  Do you feel strongly about this?